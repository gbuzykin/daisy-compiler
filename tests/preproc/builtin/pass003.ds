const s = __file__;
