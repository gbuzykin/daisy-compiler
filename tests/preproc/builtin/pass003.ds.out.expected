./preproc/builtin/pass003.ds:1:1: debug: token
 1 | const s = __file__;
   | ^~~~~
./preproc/builtin/pass003.ds:1:7: debug: id: s
 1 | const s = __file__;
   |       ^
./preproc/builtin/pass003.ds:1:9: debug: token
 1 | const s = __file__;
   |         ^
./preproc/builtin/pass003.ds:1:11: debug: string: "./preproc/builtin/pass003.ds"
 1 | const s = __file__;
   |           ^~~~~~~~
./preproc/builtin/pass003.ds:1:7: debug: defining constant `s`
 1 | const s = __file__;
   |       ^
./preproc/builtin/pass003.ds:1:19: debug: token
 1 | const s = __file__;
   |                   ^
./preproc/builtin/pass003.ds: info: warnings 0, errors 0
